backend/src/components/permissionUtils.js

```diff
@@ -290,6 +315,26 @@ function findSdcDistrictCollectionID_params(req, res, next) {
   return next();
 }
 
+function findSInstituteTypeCollectionID_body(req, res, next) {
+  if(req.session.activeInstituteType === 'DISTRICT') {
+    res.locals.requestedInstituteType = 'DISTRICT';
+    res.locals.requestedSdcDistrictCollectionID = req.body.sdcDistrictCollectionID;
+    return next();
+  } else {
+    res.locals.requestedInstituteType = 'SCHOOL';
+    res.locals.requestedSdcSchoolCollectionID = req.body.sdcSchoolCollectionID;
+    return next();
+  }
+}
+
+function checkStudentBelongsInCollection(req, res, next) {
```

Comment:
Could we alter or rename this function? I think it would be best to have the "check" functions send HTTP rejection responses.

Reply:
Pushed a change for this. I haven't tested it since there is no collection data in the tables